__mocks__/mock_node.js

```diff
@@ -28,21 +32,40 @@ if (process.env.CONFIG_NAME === 'development') {
       ]
   };
 }
-else {
-  //
-  // If this is a live setting, we only mock the bank API
-  // (and gmail?)
-  //
-  console.warn('--- Injecting Minimal TC mocks ---');
-  var resolveFilename = Module._resolveFilename;
-  const mockedModules = ["@thecointech/rbcapi", "@thecointech/store", "@thecointech/email", "@thecointech/market-status"];
-  // In dev:live, we do not call off the machine
-  if (process.env.CONFIG_NAME === 'devlive')
-    mockedModules.push("googleapis", "google-auth-library");
 
-  Module._resolveFilename = (request, parent, isMain) => {
-    return (mockedModules.find(m => request.startsWith(m)) && !parent.path.includes('__mocks__'))
-      ? `${__dirname}/${request}.ts`
-      : resolveFilename(request, parent, isMain)
-  }
+////////////////////////////////////////////////////////////////
+
+switch (process.env.CONFIG_NAME) {
+  case 'development':
+    //
+    // If running in dev mode, use all available mocks
+    //
+    console.warn('--- Injecting All TC mocks ---');
+    mockAll();
+    break;
+  case 'devlive':
+    //
+    // Dev live is internally connected, but all external connections are mocked
+    //
+    console.warn('--- Injecting external TC mocks ---');
+    mockModules(
+      "googleapis",
+      "google-auth-library",
+      "@thecointech/rbcapi",
+      "@thecointech/store",
+      "@thecointech/email",
+      "@thecointech/market-status"
+    );
+    break;
+  case 'prodtest':
+    //
+    // ProdTest is fully live (just no real money)
+    //
+    console.warn('--- Injecting Bank mocks ---');
+    mockModules("@thecointech/rbcapi");
```

Comment:
Added mocks for prod:test - which is only the bank (everything else is running production code)